tests/integration/incremental_indexing/test_unique_indexing.py

```diff
@@ -4,8 +4,7 @@
 from jina.executors.indexers.keyvalue import BinaryPbIndexer
 from jina.executors.indexers.vector import NumpyIndexer
 from jina.flow import Flow
-from tests.integration.incremental_indexing import random_workspace, get_duplicate_docs
```

Comment:
It seems for pytest to accept this fixture, you should import `random_workspace` as well. I am voting for rather using 
```
@pytest.mark.parametrize('random_workspace_name', ['JINA_TEST_INCREMENTAL_INDEX_WORKSPACE'])
    def test_unique_indexing_vecindexers_before(test_metas):
    ...
```
here, since this is the way we started to use elsewhere as well.

Reply:
You may chose a dedicated workspace per test or the same for all of them.